index.js

```diff
@@ -3,7 +3,7 @@
 var BufferStreams = require('bufferstreams');
 var htmlmin = require('html-minifier');
 var gutil = require('gulp-util');
-var objectAssign = require('object-assign');
```

Comment:
You're looking at it from your perspective and the tools you use. From my perspective, object-assign creates an unnecessary dependency in my own dependency chain, as well users that use any of my projects. I don't want object-assign used, so if you don't want extend-shallow then I suggest the compromise of creating a shallow clone function in place. It's for one line of code, so this is trivial IMHO.